Formula/b/batt.rb

```diff
@@ -17,27 +17,27 @@ class Batt < Formula
   depends_on :macos
 
   def install
-    # batt does not provide separate control for AllowNonRootAccess.
-    # Normally it's changed during service file installation,
-    # but brew service supersedes that, leaving a daemon refusing to
-    # talk to anything non-root. Changing the defaults here.
-    inreplace "conf.go", "AllowNonRootAccess:      false,",
-                         "AllowNonRootAccess:      true,"
-    # Limit config path to Homebrew prefix. The socket remains in /var/run
-    # to deter non-root invocation of daemon
-    inreplace ["conf.go", "README.md"], "/etc/batt.json", etc/"batt.json"
-    # Due to local changes version tag would show v0.2.1-dirty
-    system "make", "VERSION=v#{version}"
+    # Point to the correct path for the binary
     inreplace "hack/cc.chlc.batt.plist", "/path/to/batt", bin/"batt"
```

Comment:
We should probably point the `plist` at a stable path:
```suggestion
    inreplace "hack/cc.chlc.batt.plist", "/path/to/batt", opt_bin/"batt"
```

Reply:
Sure